src/components/general/tooltip.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Tooltip } from 'reactstrap'
+import { css } from 'react-emotion'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+export default class TooltipWrapper extends React.Component {
```

Comment:
Should this have a different name? The file's called `tooltip`

Reply:
any suggestions? We already have <Tooltip> being imported from reactsrap :/